### DISCUSSION TITLE
"You have unread notifications" but inbox is empty

### DISCUSSION THREAD

**Discussion by Antiarchitect** [community/community#114987] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

The problem is blue dot indicates I have some unread notifications but the list is empty
![image](https://github.com/community/community/assets/50209/cd2545c4-0f41-4c45-b045-da56c1201327)
![image](https://github.com/community/community/assets/50209/4652594f-f5ed-4e45-a978-a39e32598b26)

**Comment by afpro** [comment]
me too, i trigger this by subscribe a PR, but notification list is empty.

**Reply by Impa10r** [reply]
Same

![Untitled](https://github.com/community/community/assets/101550606/0bd54497-04d7-424f-b5f8-1139c604b035)

**Comment by marybucy** [comment]
Although your inbox is empty, you have unread notifications awaiting your attention.
If anyone want to know about [epoxy floor experts](https://www.gnugarage.com/) visit here

**Comment by CherryDT** [comment]
Same issue here:

![image](https://github.com/community/community/assets/6010463/3a20daab-4c2f-4360-885d-1ca2d2944263)

**Comment by Antiarchitect** [comment]
Apparently this been fixed. Having empty inbox.

**Comment by william5667** [comment]
It sounds like a glitch or sync issue. Try refreshing the page, clearing your cache, or[  bath mats](https://cotonmode.co.uk/collections/bath-mats) logging out and back in. If the issue persists, contact support.
